polars_hash/polars_hash/__init__.py

```diff
@@ -158,10 +148,9 @@ def to_coords(self) -> pl.Expr:
 
     def from_coords(self, len: int | str | pl.Expr = 12) -> pl.Expr:
         """Takes Struct with latitude, longitude as input and returns utf8 hash using geohash."""
-        len_expr = wrap_expr(parse_as_expression(len))
```

Comment:
This one I believe still needs to used because input can also be int or str,

Reply:
wouldn't Polars take care of this here

https://github.com/pola-rs/polars/blob/b957d53aa6d835e5c456bc98943cde960658633d/py-polars/polars/expr/expr.py#L9680-L9683

?

Reply:
Ah I did not know that :) that's nice!